uk.co.compucorp.civicrm.hrleaveandabsences/sql/auto_install.sql

```diff
@@ -372,3 +372,25 @@ CREATE TABLE `civicrm_hrleaveandabsences_contact_work_pattern` (
     CONSTRAINT FK_civicrm_hrlaa_contact_work_pattern_contact_id FOREIGN KEY (`contact_id`) REFERENCES `civicrm_contact`(`id`) ON DELETE CASCADE,
     CONSTRAINT FK_civicrm_hrlaa_contact_work_pattern_pattern_id FOREIGN KEY (`pattern_id`) REFERENCES `civicrm_hrleaveandabsences_work_pattern`(`id`) ON DELETE CASCADE
 )  ENGINE=InnoDB DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci  ;
+
+-- /*******************************************************
+-- *
+-- * civicrm_hrleaveandabsences_sickness_request
+-- *
+-- * Sickness Request
+-- *
+-- *******************************************************/
+CREATE TABLE `civicrm_hrleaveandabsences_sickness_request` (
+
+
+  `id` int unsigned NOT NULL AUTO_INCREMENT  COMMENT 'Unique SicknessRequest ID',
+  `leave_request_id` int unsigned NOT NULL   COMMENT 'FK to LeaveRequest',
+  `reason` varchar(512) NOT NULL   COMMENT 'One of the values of the Sickness Reason option group',
+  `required_documents` varchar(10)    COMMENT 'A list of values from the LeaveRequestRequiredDocument option group'
+  ,
+  PRIMARY KEY ( `id` )
+
+
+  ,          CONSTRAINT FK_civicrm_hrleaveandabsences_sickness_request_leave_request_id FOREIGN KEY (`leave_request_id`) REFERENCES `civicrm_hrleaveandabsences_leave_request`(`id`) ON DELETE CASCADE
+)  ENGINE=InnoDB DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci  ;
```

Comment:
can you please improve the indentation of this? Check the other create table statements to see how they were formatted.